app/views/teaching_events/about_git_events.html.erb

```diff
@@ -28,8 +28,16 @@
       <li>find out how to apply for teacher training</li>
       <li>explore what funding is available</li>
     </ul>
-  </section>
-
+<p>Watch our video to see what happens at a Get Into Teaching event and how it could help you make a decision.</p>
+<section class="col col-full-content youtube-videos">
```

Comment:
@martyn-w Is a section needed here? It is creating a lot of white space below the video.

If it is needed, let's leave as is!

Reply:
That section isn't necessary - I've just pushed a small update which removes it.